src/Microsoft.DotNet.RemoteExecutor/src/RemoteExecutor.cs

```diff
@@ -16,6 +17,8 @@ namespace Microsoft.DotNet.RemoteExecutor
 {
     public static partial class RemoteExecutor
     {
+        internal const string REMOTE_EXECUTOR_ENVIRONMENTAL_VARIABLE = "DOTNET_REMOTE_EXECUTOR";
```

Comment:
nit: to be more consistent with `DOTNET_REMOTEEXECUTOR_SUPPORTED`
```suggestion
        internal const string REMOTE_EXECUTOR_ENVIRONMENTAL_VARIABLE = "DOTNET_REMOTEEXECUTOR";
```